llama_stack/apis/common/type_system.py

```diff
@@ -46,6 +46,21 @@ class CustomType(BaseModel):
     validator_class: str
 
 
+class ChatCompletionInputType(BaseModel):
```

Comment:
we should kill CustomType in that case

Reply:
let me comment it out together in https://github.com/meta-llama/llama-stack/pull/323